twake/backend/node/src/services/files/web/routes.ts

```diff
@@ -34,14 +34,20 @@ const routes: FastifyPluginCallback<{ service: FileServiceAPI }> = (
   fastify.route({
     method: "GET",
     url: `${filesUrl}/:id`,
-    preValidation: [fastify.authenticate],
+    preValidation: [], //[fastify.authenticate],
     handler: fileController.get.bind(fileController),
   });
 
+  fastify.route({
+    method: "GET",
+    url: `${filesUrl}/:id/thumbnails/:index`,
+    handler: fileController.thumbnail.bind(fileController),
+  });
+
   fastify.route({
     method: "DELETE",
     url: `${filesUrl}/:id`,
-    preValidation: [fastify.authenticate],
+    preValidation: [], //[fastify.authenticate],
```

Comment:
You must reuse all the [fastify.authenticate] (to re-protect your routes)
Except for the thumbnails where you can keep it like this and the download that you can also unprotect.